### PR TITLE
govc: add vm.customize -dns-suffix flag

### PR DESCRIPTION
**Pull request by dougm**
Issue #2293

### CODE REVIEW

File: govc/test/vm.bats [hunk 1]
```diff
@@ -1036,7 +1036,7 @@ load test_helper
   run govc vm.customize -vm DC0_H0_VM0 -type Linux
   assert_failure # no -ip specified
 
-  run govc vm.customize -vm DC0_H0_VM0 -ip 10.0.0.45 -netmask 255.255.0.0 -type Linux
+  run govc vm.customize -vm DC0_H0_VM0 -ip 10.0.0.45 -netmask 255.255.0.0 -type Linux -dns-server 1.1.1.1 -dns-suffix example.com
```
Comment (embano1) [inline govc/test/vm.bats:1039]: Can you provide a test which show/validates multiple (ie list)?

File: govc/vm/customize.go [hunk 2]
```diff
@@ -69,8 +70,10 @@ func (cmd *customize) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.gateway = nil
 	f.Var(&cmd.netmask, "netmask", "Netmask")
 	cmd.netmask = nil
-	f.Var(&cmd.dnsserver, "dns-server", "DNS server")
+	f.Var(&cmd.dnsserver, "dns-server", "DNS server list")
 	cmd.dnsserver = nil
+	f.Var(&cmd.dnssuffix, "dns-suffix", "DNS suffix list")
+	cmd.dnssuffix = nil
```
Comment (embano1) [inline govc/vm/customize.go:76]: Is that needed?

Comment (dougm) [reply govc/vm/customize.go:76]: Not normally, but @gressd has some code that embeds some govc commands (including this one), needs to clear the array after each call.

File: govc/test/vm.bats (hunk 1 above)
Comment (dougm) [reply govc/test/vm.bats:1039]: In real vCenter these values would land inside the VM, files such as /etc/resolv.conf, /etc/hosts, etc.  The API (and hence vcsim) doesn't have access to them.

**Review by embano1 — APPROVED** [review]
LGTM

**Pull request merged**